### PR TITLE
Workspace title

### PR DESCRIPTION
**Pull request by chenilim**
#### Summary
* Show a hint when there is no active view
* Show the workspace name in the sidebar, and in the hint if available

Notes:
* Updated auth API to return workspace info
* Just title for now, but we can extend this to more info later

### CODE REVIEW

File: webapp/src/components/workspace.tsx [hunk 1]
```diff
@@ -21,16 +24,36 @@ type Props = {
     readonly: boolean
 }
 
-const Workspace = React.memo((props: Props) => {
-    const {boardTree, setSearchText, workspaceTree, showBoard, showView, setLanguage} = props
+function centerContent(props: Props) {
```
Comment (jespino) [inline webapp/src/components/workspace.tsx:27]: In fact, this is a functional component, we should make it uppercase and use as a component bellow.

**Review by jespino — APPROVED** [review]
A tiny detail, but otherwise looks good to me.

**Comment by jespino** [comment]
I guess this needs changes in the enterprise repo too.

**Comment by chenilim** [comment]
Thanks Jesus! The enterprise changes are already in. I'll check this in and clean up the functional component after.

**Pull request merged**